Added german language + new contains rule

mintalicious opened the pull request:
After closing the last PR, I created a new one which contains the german language file, a new contains rule and little update of the required rule, which now checks the $value to not be empty instead of only not being null.

mintalicious committed:
Added german language + new contains rule

pdscopes commented on src/Validate.php:
```diff
@@ -547,6 +548,29 @@ public static function notIn(Validator $validator, $data, $pattern, $rule, $para
     }
 
+    /**
+     * contains-only:<value>(,<value>)*
```
You forgot to update this part of the comment
```suggestion
     * contains:<value>(,<value>)*
```

pdscopes commented on src/Validate.php:
```diff
@@ -547,6 +548,29 @@ public static function notIn(Validator $validator, $data, $pattern, $rule, $para
+     * @param string $rule
+     * @param array  $parameters
+     */
+    public static function contains(Validator $validator, $data, $pattern, $rule, $parameters)
+    {
+        foreach (Validator::getValues($data, $pattern) as $attribute => $value) {
+            if (null === $value || empty($value)) {
+                continue;
+            }
+            if (is_countable($value) && count($parameters) == count(array_intersect($value, $parameters))) {
+                continue;
+            }
+
+            $validator->addError($attribute, $rule, [':values' => implode(', ', $parameters)]);
+        }
+    }
```
Please could you add an integration test to cover this new validation?

See `tests/Integration/ValidateContainsOnlyTest.php`  as an example.

pdscopes reviewed (changes requested):
Thanks for doing this so quickly, I have two comments for you to please address and then I will merge and release v3.

mintalicious committed:
Fixed PHPDoc for contains rule + added integration test

mintalicious commented:
I fixed the PHPDoc for contains() and added an integration test file as suggested.
Please do not tag it as v3 - the PR for changing the return value of validate() is yes to come.

mintalicious commented:
Btw: Thx for this nice little validation library ;-)

pdscopes commented:
You are very welcome, I use it personally and I'm happy to share it!

pdscopes commented:
Thanks again, this has now been release in [v2.7.0](https://github.com/pdscopes/php-form-validator/releases/tag/v2.7.0)
